serialization/src/main/java/org/nervos/ckb/type/fixed/Option.java

```diff
@@ -0,0 +1,29 @@
+package org.nervos.ckb.type.fixed;
+
+import org.nervos.ckb.type.base.FixedType;
+import org.nervos.ckb.type.base.Type;
+
+/** Copyright © 2019 Nervos Foundation. All rights reserved. */
+public class Option extends FixedType<Type> {
```

Comment:
I recommend using a more specific name, like `BytesOpt`, `option` in serialization RFC is dynamic size, and is a `Composite Types`. https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0008-serialization/0008-serialization.md#option

Reply:
`BytesOpt` can be implemented by`new Option(new Bytes(value))`, and `Option` here can be dynamic.

You can see https://github.com/nervosnetwork/ckb-sdk-java/blob/update-witnesses/core/src/main/java/org/nervos/ckb/utils/Serializer.java#L94-L100

Reply:
why put option under the `/fixed/`

Reply:
Updated. 
* https://github.com/nervosnetwork/ckb-sdk-java/pull/202/commits/031038affd37b18242a6fe0339dfc9736bc3c3f7

* https://github.com/nervosnetwork/ckb-sdk-java/pull/202/commits/55ad71a8808a9daae619bebb16844617160655a1